core/template/scenario/remove_inat.default.html

```diff
@@ -0,0 +1,33 @@
+<!--template/scenario/remove_inat.default-->
+<div class="input-group input-group-sm" style="width: 100%">
+  <span class="input-group-addon roundedLeft">{{Scénario}}</span>
+  <select class="expressionAttr form-control input-sm roundedRight"  data-l1key="options" data-l2key="scenario_id" data-cmd_id="#id#" data-uid="#uid#"></select>
+</div>
+<script>
+  jeedom.scenario.allOrderedByGroupObjectName({
+    asGroup: 0,
+    error: function (error) {
+      jeedomUtils.showAlert({message: error.message, level: 'danger'});
+    },
+    success: function (scenarios) {
+      var select = document.querySelector('.expressionAttr[data-uid="#uid#"][data-l1key="options"][data-l2key="scenario_id"]')
```

Comment:
Consider using template literals for string concatenation to improve readability. [medium]

Reply:
Note that this file is originally a copy of `core/template/scenario/scenario.default.html`.
Line 13 is already a literal string, no templating nor concatenation required.
Do you agree?
Anyway, I changed the following lines in commit 33b3f56 to shorten and optimize querySelectors.